### PR TITLE
Updates the Xcode settings.

### PR DESCRIPTION
**Pull request by sergiocampama**
1. To use the previous version of Swift instead of the new one.
2. To update the deployment versions, which disables the inclusion of arclite and avoids a duplicate symbol error.

Tested with both 7.3.1 and 8.0 beta Xcodes

https://github.com/google/protobuf/issues/1679

@thomasvl fyi

### CODE REVIEW

File: objectivec/ProtocolBuffers_OSX.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -826,7 +828,7 @@
 				GCC_WARN_UNUSED_PARAMETER = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				GENERATE_PROFILING_CODE = NO;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```
Comment (thomasvl) [inline objectivec/ProtocolBuffers_OSX.xcodeproj/project.pbxproj]: Why is this needed, this effectively means our min version is being moved up.

Comment (thomasvl) [inline objectivec/ProtocolBuffers_OSX.xcodeproj/project.pbxproj]: Or is this only on the tests?  In which case the cl comment should get tweak to make it clear it isn't moving the baseline up and just for the tests because of the Swift test.

Comment (sergiocampama) [inline objectivec/ProtocolBuffers_OSX.xcodeproj/project.pbxproj]: Yeah, it's because of the swift test, as using a lower value includes the arclite library twice (once from the default toolchain and once from the swift 2.3 toolchain) and errors with duplicate symbols... Although I would expect this to be a bug in the xcode beta... should we wait for new releases to see if it gets fixed?

Comment (thomasvl) [inline objectivec/ProtocolBuffers_OSX.xcodeproj/project.pbxproj]: Yea, lets close this PR then without landing things and just keep tabs on the betas to see where things end up.

**Comment by sergiocampama** [comment]
skipping for now, until xcode8 rolls out in GM
